sdk/src/Services/S3/Custom/Transfer/TransferUtilityUploadDirectoryRequest.cs

```diff
@@ -50,6 +50,9 @@ public class TransferUtilityUploadDirectoryRequest : BaseUploadRequest
         MetadataCollection metadataCollection;
         ServerSideEncryptionMethod encryption;
         string serverSideEncryptionKeyManagementServiceKeyId;
+        private ServerSideEncryptionCustomerMethod serverSideCustomerEncryption;
```

Comment:
Minor, but could you name the variable `serverSideCustomerEncryptionMethod` to match the type?

Given all the encryption properties with slightly different names, just want to help us keep them straight.

Reply:
I have copied this part from TransferUtilityUploadRequest. I have done a little research on naming of this parameters, and it would be more straight to stick to the current name, what do you think?
Maybe renaming all of them to one name is a different issue?
| Name                               | Class                                    |
|------------------------------------|------------------------------------------|
| serverSideCustomerEncryption       | CopyObjectRequest.cs                     |
| serverSideCustomerEncryption       | CopyObjectResponse.cs                    |
| serverSideCustomerEncryption       | CopyPartRequest.cs                       |
| serverSideCustomerEncryption       | GetObjectMetadataRequest.cs              |
| serverSideCustomerEncryption       | GetObjectRequest.cs                      |
| serverSideCustomerEncryption       | GetPreSignedUrlRequest.cs                |
| serverSideCustomerEncryption       | InitiateMultipartUploadRequest.cs        |
| serverSideCustomerEncryption       | InitiateMultipartUploadResponse.cs       |
| serverSideCustomerEncryption       | PutObjectRequest.cs                      |
| serverSideCustomerEncryption       | PutObjectResponse.cs                     |
| serverSideCustomerEncryption       | UploadPartRequest.cs                     |
| serverSideCustomerEncryption       | BaseDownloadRequest.cs                   |
| serverSideCustomerEncryption       | TransferUtilityUploadRequest.cs          |
| serverSideCustomerEncryption       | TransferUtilityUploadDirectoryRequest.cs |
| serverSideEncryptionCustomerMethod | GetObjectMetadataResponse.cs             |
| serverSideEncryptionCustomerMethod | GetObjectResponse.cs                     |
| sSECustomerAlgorithm               | WriteGetObjectResponseRequest.cs         |

Reply:
Ah, I didn't realize it was already that inconsistent. No worries then.